docs/maintainers/README.md

```diff
@@ -20,14 +20,10 @@ They have [write access](https://help.github.com/articles/repository-permission-
 
 ## Current Repository Maintainers
 
-* Sergei Vorobev ([vors](https://github.com/vors))
-* Jason Shirk ([lzybkr](https://github.com/lzybkr))
-* Dongbo Wang ([daxian-dbw](https://github.com/daxian-dbw))
-* Travis Plunk ([TravisEz13](https://github.com/TravisEz13))
-* Mike Richmond ([mirichmo](https://github.com/mirichmo))
-* Andy Schwartzmeyer ([andschwa](https://github.com/andschwa))
-* Aditya Patwardhan ([adityapatwardhan](https://github.com/adityapatwardhan))
-* Ilya Sazonov ([iSazonov](https://github.com/iSazonov))
+- Dongbo Wang ([daxian-dbw](https://github.com/daxian-dbw))
```

Comment:
Ironically, in this case PR Summary does not entirely correctly summaries the changes. :)
PR Summary += "... and updating maintainer list"